packages/@aws-cdk/aws-lambda/lib/log-retention.ts

```diff
@@ -1,116 +1,31 @@
-import * as path from 'path';
-import * as iam from '@aws-cdk/aws-iam';
 import * as logs from '@aws-cdk/aws-logs';
 import * as cdk from '@aws-cdk/core';
-import { Code } from './code';
-import { Runtime } from './runtime';
-import { SingletonFunction } from './singleton-lambda';
 
 /**
- * Construction properties for a LogRetention.
+ * Retry options for all AWS API calls.
+ *
+ * @deprecated use `LogRetentionRetryOptions` from '@aws-cdk/aws-logs' instead
  */
-export interface LogRetentionProps {
-  /**
-   * The log group name.
-   */
-  readonly logGroupName: string;
-
-  /**
-   * The number of days log events are kept in CloudWatch Logs.
-   */
-  readonly retention: logs.RetentionDays;
-
-  /**
-   * The IAM role for the Lambda function associated with the custom resource.
-   *
-   * @default - A new role is created
-   */
-  readonly role?: iam.IRole;
-
-  /**
-   * Retry options for all AWS API calls.
-   *
-   * @default - AWS SDK default retry options
-   */
-  readonly logRetentionRetryOptions?: LogRetentionRetryOptions;
+export interface LogRetentionRetryOptions extends logs.LogRetentionRetryOptions {
```

Comment:
This isn't used anywhere now, but I assume it technically needs to exist for backwards compatibility, yes?

Reply:
Yeah... If anyone uses it in their own code